docs/dm/assets.md

```diff
@@ -56,4 +64,31 @@ When you drop an asset on the board that has templates associated with it, you'l
 </video>
 
 This can be very handy to reuse some tokens you frequently use to prepopulate things like HP and light sources.
-This will also be key to sharing sets with other players.
+This will also be key to sharing sets with other players (see [downloading files](#downloading-files) above).
+
+### Variants
+
+:::warning Experimental!
+This feature is experimental! Some cornercases are expected to appear, so please [report to the issue tracker at GitHub](https://github.com/Kruptein/PlanarAlly/issues) if you encounter any bugs!
```

Comment:
You mentioned this yourself, but the link doesn't look great,
We can however change the styling on this to our liking, I'll post some info on how to later